vexim/config/header.php

```diff
@@ -1,24 +1,25 @@
 <?php
-  if (isset($_SESSION['domain_id'])) {
-    $domheaderquery = "SELECT enabled FROM domains WHERE domains.domain_id=:domain_id";
-    $domheaderresult = $dbh->prepare($domheaderquery);
-    $domsuccess = $domheaderresult->execute(array(':domain_id'=>$_SESSION['domain_id']));
-    if ($domsuccess) { $domheaderrow = $domheaderresult->fetch(); }
-    $usrheaderquery = "SELECT enabled FROM users WHERE localpart=:localpart AND domain_id=:domain_id";
-    $usrheaderresult = $dbh->prepare($usrheaderquery);
-    $usrsuccess = $usrheaderresult->execute(array(':localpart'=>$_SESSION['localpart'], ':domain_id'=>$_SESSION['domain_id']));
-    if ($usrsuccess) { $usrheaderrow = $usrheaderresult->fetch(); }
-  }
-
-  print "<div id=\"Header\"><p><a href=\"https://github.com/avleen/vexim2\" target=\"_blank\">" . _("Virtual Exim") . "</a> ";
+  print "<div id=\"Header\"><p><a href=\"https://github.com/vexim/vexim2\" target=\"_blank\">" . _("Virtual Exim") . "</a> ";
   if (isset($_SESSION['domain'])) {
     print     "-- " . $_SESSION['domain'] . " ";
   }
-  if (isset($_SESSION['domain_id'])) {
-    if ($domheaderrow['enabled'] === "0") {
-      print   _("-- domain disabled (please see your administrator).");
-    } else if ($usrheaderrow['enabled'] === "0") {
-      print   _("-- account disabled (please see your administrator).");
+  if (isset($_SESSION['domain_id'])) { // Check if accound or domain haven't been disabled
+    $headerquery = "SELECT domains.enabled AS domain, users.enabled AS user FROM users,domains
+                    WHERE users.localpart=:localpart AND domains.domain_id=:domain_id AND users.domain_id=domains.domain_id;";
+    $headerresult = $dbh->prepare($headerquery);
+    $headersuccess = $headerresult->execute(array(':localpart'=>$_SESSION['localpart'], ':domain_id'=>$_SESSION['domain_id']));
+    if ($headersuccess) {
+      $headerrow = $headerresult->fetch();
+      if ($headerrow['domain'] === "0") {
+        unset($_SESSION['domain_id']);
```

Comment:
I suppose it would be better to do here what `logout.php` does, only the `header()` call should remain different. Perhaps we should even have it always done in a single place, e.g. in `functions.php`?